src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -48,7 +48,13 @@
         "symfony/twig-bridge": "^5.4|^6.0",
         "symfony/validator": "^5.4|^6.0",
         "symfony/yaml": "^5.4|^6.0",
-        "twig/twig": "^2.13|^3.0.4"
+        "twig/twig": "^2.13|^3.0.4",
+        "web-token/jwt-checker": "^3.1",
+        "web-token/jwt-signature-algorithm-hmac": "^3.1",
+        "web-token/jwt-signature-algorithm-ecdsa": "^3.1",
+        "web-token/jwt-signature-algorithm-rsa": "^3.1",
+        "web-token/jwt-signature-algorithm-eddsa": "^3.1",
+        "web-token/jwt-signature-algorithm-none": "^3.1"
```

Comment:
It would be nice to somehow reduce the number of packages to require, this looks over-splitted to me. /cc @Spomky (can also be adressed later, but it's critical imho)

Reply:
Btw do we actually need those? The code paths where they are used don't seem to be tested (an update to SecurityBundle's MainConfigurationTest asserting that the expected exceptions are thrown and that the possible configs result in the proper service definitions would be welcome though)

Reply:
It is tested in `src/Symfony/Bundle/SecurityBundle/Tests/Functional/AccessTokenTest.php`.

Either I remove this dependency, remove the `AccessTokenTest` related test, and update the `MainConfigurationTest` checking any related exception; or I keep this dependency and the `AccessTokenTest` related test, but can't update accordingly the `MainConfigurationTest`. IHMO, I prefer the first solution which ensures the feature is properly working.

Reply:
I've updated these dependencies using [`web-token/signature-pack`](https://packagist.org/packages/web-token/signature-pack) instead (same in root composer.json file)

Reply:
BTW @chalasr, do I need this dev-dependency here, as it's already added in the composer.json at the root path (symfony/symfony)?